src/ansiCommandStream.js

```diff
@@ -37,11 +37,23 @@ const theme = {
   },
 };
 
+// use this function to adjust arguments to yarn or npm
+const adjustArguments = arg => {
+  switch (arg) {
+    case 'remove':
+      return 'uninstall';
+    case 'add':
+      return 'i';
```

Comment:
Needs to be `[“install”,“—save”]`, right?

Reply:
"npm i" is the abbreviation of "npm install --save"

Reply:
Oh I forgot about that shortcut. Seems safe but the longer version will work better with older versions